superagent/superagent.d.ts

```diff
@@ -77,6 +77,7 @@ declare module "superagent" {
       notAcceptable: boolean;
       notFound: boolean;
       forbidden: boolean;
+      xhr: XMLHttpRequest;
```

Comment:
`XMLHttpRequest` exists in lib.d.ts though. Can you provide a minimum repro sample code and open a separate issue in this repo for better tracking? Thanks!